src/lib/components/works_for_you/notification.tsx

```diff
@@ -10,14 +10,15 @@ import colors from "../../../data/colors"
 
 interface Props extends RelayProps {}
 
-class Notification extends React.Component<Props, any> {
+export class Notification extends React.Component<Props, any> {
   render() {
     const notification = this.props.notification
 
     return (
       <View style={styles.container}>
         <View style={styles.header}>
-          <View>
+          { notification.image && <Image source={{uri: notification.image.resized.url}} style={styles.artistAvatar}/> }
```

Comment:
Yeah I was thinking about how best to handle missing images - right now it looks like this if an artist doesn't have an avatar: 

![simulator screen shot mar 23 2017 4 37 29 pm](https://cloud.githubusercontent.com/assets/2712962/24255935/1caa3b14-0fe7-11e7-96ce-cb68078337ef.png)

And it would look like this with a light gray placeholder:
![simulator screen shot mar 23 2017 4 41 53 pm](https://cloud.githubusercontent.com/assets/2712962/24256161/ac8750fa-0fe7-11e7-8dd8-bb4cc7571138.png)

@katarinabatina, which do you prefer for artists that don't have avatar images (or for whom the image link is faulty)?